authors/apps/report/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from django.utils.translation import pgettext_lazy as _
+from django.contrib.auth import get_user_model
+
+
+class Report(models.Model):
+    """
+    Store reported articles for Admin to review
+    """
+    reporter = models.ForeignKey(
+        get_user_model(),
+        on_delete=models.CASCADE,
+        null=True,
+        blank=True,
+        default=None
+    )
+    title = models.TextField(
+        _('Report field', 'title'),
+        blank=True,
```

Comment:
Why are you allowing blank and null value here??

Reply:
See above.  Repeat comment